catroidTest/src/org/catrobat/catroid/test/physics/PhysicsSpriteCloneTest.java

```diff
@@ -188,7 +187,7 @@ public void testSpriteClonePhysicsLookAndPhysicsObject() {
 		sprite.addScript(startScript);
 
 		PhysicsWorld physicsWorld = project.getPhysicsWorld();
-		sprite.look = new PhysicsLook(sprite, physicsWorld);
+		sprite.look = new Look(sprite);
```

Comment:
Every Sprite has a Look-Object. A PhysicsLook is generated at the first "stage-entry". So after that first "state-entry" there is a PhysicsLook in the sprite. If we copy the sprite the PhysicsLook will convert back to a Look (with the function you mentioned). So it is back in the "initial state" and will be converted into a PhysicsLook again at the first "stage-entry".
So there is no real need for a copyPhysicsLookForSprite Methode ;-)

Pls correct me if I don't concern any important point. :-)